### DISCUSSION TITLE
Welcome to djot Discussions!

### DISCUSSION THREAD

**Discussion by jgm** [jgm/djot#19] [Announcements]
This is a place to ask questions and offer feedback about djot.

**Comment by vassudanagunta** [comment]
My post of djot on Hacker News made it to the front page and has sparked a growing discussion over there: https://news.ycombinator.com/item?id=33867636

**Comment by Firefingertipcoder** [comment]
Hi,I am New here,
My Name is Bolt
I have learned HTML (only HTML)
And Has a Complete grip on it .
Hope I can be friends with you all
And Please Share me a brief detail explanation of the project I hope I can be some use of you.......😃
